addons/pos_hr/models/hr_employee_public.py

```diff
@@ -18,4 +18,4 @@ def read(self, fields):
             if pos_blackbox_be_installed and not has_hr_user_group:
                 fields.remove('insz_or_bis_number')
 
-        return super().read(fields)
+        return super(HrEmployeePublic, self).read(fields=fields, load=load)
```

Comment:
```suggestion
        return super().read(fields=fields, load=load)
```

Super args aren't needed.

Reply:
Thanks for your review @davidmonnom 
Another fix is merging.